packages/types/Calendar.d.ts

```diff
@@ -188,6 +188,8 @@ export interface CalendarEvent {
 
   // It just has responses to only the user fields. It allows to easily iterate over to show only user fields
   userFieldsResponses?: CalEventResponses | null;
+  // It just has responses to only the user-owner-secret fields. It allows to easily iterate over to show only user fields
+  userOwnerSecretFieldsResponses?: CalEventResponses | null;
```

Comment:
We shouldn't introduce a new property here that could just be an attribute in userFieldsResponse. I think we should have a prop `isSecretQuestion` per field response in userFieldsResponses.